setup.py

```diff
@@ -23,7 +23,7 @@ def has_ext_modules(foo) -> bool:
 
 setuptools.setup(
     name='cohere',
-    version='1.3.8',
+    version='1.3.9',
```

Comment:
Do we follow semantic versioning here, i.e. should this change (adding the extract API) be considered a minor update (new feature) rather than a patch?

I wonder if we should consider a more structured approach to SDK version updates. e.g. https://python-semantic-release.readthedocs.io/en/latest/ compute the version automatically based on commit messages, and generate the changelog appropriately too. Something for the future.

Reply:
I proposed semantic versioning too! @adrian-cohere is looking at a bunch of improvements we can make to our SDKs, we're going to start a list soon.

For this case, this is a V0 API release, mostly for us to play with internally, so I think a patch is maybe applicable here?

Reply:
I think it's a good idea. In practice, the risk here is low, unless some external party starts using an undocumented function, but there is no reason to add a non-prefixed version right now.